### PR TITLE
Move unicorn_killer to Gemfile

### PR DESCRIPTION
**Pull request by ghost**
I stumbled over that and though it is a good idea to use the gem file instead. There are a view changes since 2013.

There is also the possibility to kill workers by max_requests:

```
use Unicorn::WorkerKiller::MaxRequests, max_request_min, max_request_max
```

Do we need/want this? If so.. unfortunately I have no reference for some max/min values:
I found some examples with 500-600 request per worker.

related to #6269

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -5,12 +5,15 @@
 # This file is used by Rack-based servers to start the application.
 
 require ::File.expand_path("../config/environment",  __FILE__)
-require ::File.expand_path("../lib/unicorn_killer",  __FILE__)
 require ::File.expand_path("../lib/rack/internet_explorer_version", __FILE__)
 
 # Kill unicorn workers really aggressively (at 300mb)
 if defined?(Unicorn)
-  use UnicornKiller::Oom, 300 * 1024
+  require 'unicorn/worker_killer'
```
Comment (diaspora-code-review) [inline config.ru]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: config.ru [hunk 2]
```diff
@@ -5,12 +5,15 @@
 # This file is used by Rack-based servers to start the application.
 
 require ::File.expand_path("../config/environment",  __FILE__)
-require ::File.expand_path("../lib/unicorn_killer",  __FILE__)
 require ::File.expand_path("../lib/rack/internet_explorer_version", __FILE__)
 
 # Kill unicorn workers really aggressively (at 300mb)
 if defined?(Unicorn)
-  use UnicornKiller::Oom, 300 * 1024
+  require "unicorn/worker_killer"
```
Comment (denschub) [inline config.ru]: This `require` should be on top of the file.

Comment (jhass) [inline config.ru]: Not necessarily, since the `defined?(Unicorn)` makes unicorn optional with no changes besides the Gemfile.

Comment (denschub) [inline config.ru]: I was more worried about codestyle here - requires in the middle of the code look weird. But if you're fine with it, I can get over it. ;)

Comment (jhass) [inline config.ru]: It's too weird and I mean there's another require just above it stil

**Comment by denschub** [comment]
Merged. 

> Do we need/want this?

If we do, we can always add that later. :)

**Pull request merged**